app/helpers/v1/homepage_helper.rb

```diff
@@ -0,0 +1,2 @@
+module V1::HomepageHelper
```

Comment:
Apparently necessary from `rails g controller` per this thread: https://stackoverflow.com/q/49950136